fuel-coroutines/src/test/kotlin/com/github/kittinunf/fuel/coroutines/CoroutinesTest.kt

```diff
@@ -23,7 +23,7 @@ class CoroutinesTest {
     @Test
     fun testAwaitResponseSuccess() = runBlocking {
         try {
-            Fuel.get("/ip").awaitByteArrayResponse().third.fold({ data ->
```

Comment:
Don't we want to keep `awaitXX` version for our user to use? Are we gonna deprecated them and move user to use the `asyncXX` version instead? I think either way we probably want to keep the tests for `awaitXXX`

Reply:
Yeah definitely keep the tests until the await versions are removed, if ever.

Reply:
Oh Okay, I will restore awaitXX versions of tests.